nativescript-angular/platform-common.ts

```diff
@@ -224,7 +239,12 @@ export class NativeScriptPlatformRef extends PlatformRef {
         if (isLogEnabled()) {
             bootstrapLog("Angular livesync started.");
         }
-        onBeforeLivesync.next(lastBootstrappedModule ? lastBootstrappedModule.get() : null);
+
+        const lastModuleRef = lastBootstrappedModule ? lastBootstrappedModule.get() : null;
+        onBeforeLivesync.next(lastModuleRef);
```

Comment:
Do we need to emit if there's no `lastBootstrappedModule`?

Reply:
There will be no `lastBootstrappedModule` if there was an error in the initial bootstrap and `lastBootstrappedModule.get()` will return `null` if the angular app was manually killed and collected (don't think it is a common practice in NS apps, but still ...). In either case I think the `onBeforeLivesync` should emit to be in balance with the `onAfterLivesync`.